### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by ramaninsan**

### CODE REVIEW

File: local.properties [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
+## This file must *NOT* be checked into Version Control Systems,
 # as it contains information specific to your local configuration.
 #
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-sdk.dir=/Users/deepakkanyan/Library/Android/sdk
\ No newline at end of file
+#Sat Sep 03 19:12:31 EDT 2022
+sdk.dir=C\:\\Users\\raman\\AppData\\Local\\Android\\Sdk
```
Comment (deepakkanyan) [inline local.properties:8]: dont change testing comment

File: app/src/main/java/com/example/jetpackcomp/screen/CadPlaces.kt [hunk 2]
```diff
@@ -55,7 +55,7 @@ fun CadPlaces() {
                 LazyVerticalGrid(
                     contentPadding = innerPadding,
                     verticalArrangement = Arrangement.spacedBy(12.dp),
-                    cells = GridCells.Fixed(1)
```
Comment (deepakkanyan) [inline app/src/main/java/com/example/jetpackcomp/screen/CadPlaces.kt]: change fixed to 3

Comment (ramaninsan) [reply app/src/main/java/com/example/jetpackcomp/screen/CadPlaces.kt]: Done

File: local.properties (hunk 1 above)
Comment (ramaninsan) [reply local.properties:8]: Done

**Review by deepakkanyan — APPROVED** [review]
DONE

**Pull request merged**